### DISCUSSION TITLE
Feature/Idea: Field status actions/icons

### DISCUSSION THREAD

**Discussion by JoryHogeveen** [pods-framework/pods#6715] [Ideas]
It would be great to have extra field information without opening a field itself.
I imagine we could replicate the WooCommerce way of adding a "actions" column. This way we could add some quick actions (yes/no fields) as icons. Once clicked it would enable/disable that setting.

Settings I imagine would be handy:
- Required
- Restricted access (only info, no action)
- Hidden in UI
- Readable in REST
- Filter for add-ons to hook into, think of Pods SEO (enabled/disabled) and Pods Polylang Sync (enabled/disabled)

Example UI (with the above actions):
![image](https://user-images.githubusercontent.com/826148/74348320-caca6280-4db2-11ea-9de6-464e9b12af76.png)

**Comment by sc0ttkclark** [comment]
This is going to be tricky to figure out the best way forward for in the new DFV React work. I think this will have to wait until we determine a better approach that fits most common needs.
